tests/wicked/2nics/ref/t03_setup_second_card.pm

```diff
@@ -12,17 +12,20 @@
 #             Jose Lausuch <jalausuch@suse.com>
 #             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
 
-use base 'wickedbase';
-use strict;
-use warnings;
+use Mojo::Base 'wickedbase';
```

Comment:
is it necessary to add Mojo::Base in this particular PR ? generally I am totally fine with this change but I just don't want to have part of tests having `use base wickedbase` and part `use Mojo::Base 'wickedbase';` I prefer to have separate PR where we replacing ALL at one shot

Reply:
I don't have hard feelings. I would change it now and others later on, anyway if you don't like it we can do so in a separated PR.

Reply:
I don't have hard feelings too , I just have fear that using `base` and `Mojo::Base` together can bring some side effects . if you sure that all will be fine you can leave it